genesis-programs/src/lib.rs

```diff
@@ -84,16 +88,26 @@ fn get_programs(operating_mode: OperatingMode) -> Vec<(Program, Epoch)> {
         .into_iter()
         .map(|program| (program, GENESIS_EPOCH))
         .collect::<Vec<_>>(),
+        ClusterType::Devnet => vec![
+            // Programs used for testing
+            Program::BuiltinLoader(solana_bpf_loader_deprecated_program!()),
```

Comment:
it seems that `solana_bpf_loader_program!()` is yet to be added on devnet/v1.2

Reply:
That's right, now backported yet